src/assets/i18n/en.json

```diff
@@ -54,6 +54,55 @@
+    "bad-import-data-make-sure-you-selected-a-nault-address-book": "Bad import data, make sure you selected a Nault Address Book export",
+    "edit-entry": "Edit Entry",
+    "exchange-deposit-address-main-trading-account-etc": "Exchange Deposit Address, Main Trading Account, etc.",
+    "export-address-book": "Export Address Book",
+    "export-all-address-book-entries-to-a-file-or-qr-code-url": "Export all address book entries to a file or QR Code/URL.",
+    "export-as-file": "Export As File",
+    "if-you-need-a-qr-use-fewer-entries-or-shorter-aliases": "If you need a QR, use fewer entries or shorter aliases.",
+    "import-address-book": "Import Address Book",
+    "import-address-book-entries-from-a-file": "Import address book entries from a file.",
+    "import-export": "Import / Export",
+    "note-exported-address-book-will-not-be-encrypted-by-your-wal": "Note: Exported address book will NOT be encrypted by your wallet password.",
+    "reload-balances": "Reload balances",
+    "scan-the-qr-code-on-any-device-to-import-your-address-book": "Scan the QR code on any device to import your address book!",
+    "show-total-tracked-balance": "Show Total Tracked Balance",
+    "successfully-deleted-address-book-entry": "Successfully deleted address book entry",
+    "the-balance-will-be-displayed-in-the-address-book": "The balance will be displayed in the address book.",
+    "this-name-is-already-in-use-please-use-a-unique-name": "This name is already in use! Please use a unique name",
+    "this-name-is-reserved-for-wallet-accounts-without-a-label": "This name is reserved for wallet accounts without a label",
+    "track-balance": "Track Balance",
+    "track-transactions": "Track Transactions",
+    "unable-to-delete-entry": "Unable to delete entry: {{ message }}",
+    "unable-to-parse-import-data-make-sure-you-selected-the-right": "Unable to parse import data, make sure you selected the right file!",
+    "unable-to-save-entry": "Unable to save entry: {{ message }}",
+    "updated-address-book-order": "Updated address book order!",
+    "you-can-also-import-your-address-book-by-using-the-url-below": "You can also import your address book by using the URL below or download it as a file.",
+    "you-can-only-track-the-balance-of-maximum-20-addresses": "You can only track the balance of maximum 20 addresses",
+    "you-can-still-import-your-address-book-by-using-the-url": "You can still import your address book by using the URL below or download it as a file.",
+    "you-can-use-the-address-book-to-store": {
+      "1": "You can use the address book to store a label for your own accounts or for easy access to other addresses throughout the application.",
+      "2": "It's also possible to track the balance and/or transactions of any address, known as \"watch-only\"."
+    },
+    "you-do-not-have-any-address-book-entries-saved-yet": {
+      "1": "You do not have any address book entries saved yet,",
+      "2": "click here to create one",
```

Comment:
could be "2-link"

the rest looks good code changes wise, i believe

Reply:
Changed, thanks!